### ISSUE TITLE
[feature] bg1 dualclassing into a kit

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#165]
In BG1, one could dual-class into a specialist mage (the only kind of kit in that game).

e9b44bd83 added the last bits needed to support displaying and leveling them up, however we don't offer the choice to even start on that path yet. 

The code is in DualClass.py, but also the little todo about the override in IsDualClassed will need to be addressed. Basically the kit selection window (see GUICG*) needs to be displayed also in the DC process if one chooses a mage.

**Comment by arosenow** [comment]
Although tangential to this request, a fun way to let people expand the rules would be to allow penalties to the amount of XP you need to level (as opposed to just reducing XP gain). This could allow alternate rulesets which balance dual classing by requiring more XP so you aren't going to hit the cap in your second class. Given you've already got IWD2 implemented, how easy/hard it is for a user to mix leveling rules?

**Comment by lynxlynxlynx** [comment]
It would be easy, but that has nothing to do with iwd2.

**Comment by arosenow** [comment]
> It would be easy, but that has nothing to do with iwd2.

IWD2 has entirely different levelling rules to BG based on 3E, the reason I mention it is because if the engine can handle letting you pick entirely different classes with each level up and level penalties for different races as per 3E, how liberal might the engine be with applying those features across games

**Comment by lynxlynxlynx** [comment]
Not at all, since a different data format and associated rules are involved.
